lib/core/src/lib/form/components/widgets/core/form-field-validator.spec.ts

```diff
@@ -833,9 +833,9 @@ describe('FormFieldValidator', () => {
 
         it('should take into account that min value is in UTC and NOT fail validating value checking the time', () => {
             const minValueFromActivitiInput = '02-3-2018 06:00 AM';
-            const minValueSavedInForm = moment(minValueFromActivitiInput, 'DD-M-YYYY hh:mm A').utc().format();
+            const minValueSavedInForm = DateFnsUtils.parseDate(minValueFromActivitiInput, 'DD-M-YYYY hh:mm A').toISOString();
 
-            const localValidValue = '2018-3-02 06:01 AM';
```

Comment:
the test data should not be changed, it's the activity form field model... the purpose of validator is to do the magic inside and validate what's coming into...